guides/hosting/installation-updates/docker.md

```diff
@@ -26,10 +28,10 @@ The typical Dockerfile in your project would look like this:
 #syntax=docker/dockerfile:1.4
 
 ARG PHP_VERSION=8.3
-FROM ghcr.io/shopware/docker-base:$PHP_VERSION-caddy as base-image
-FROM ghcr.io/friendsofshopware/shopware-cli:latest-php-$PHP_VERSION as shopware-cli
+FROM ghcr.io/shopware/docker-base:$PHP_VERSION-caddy AS base-image
```

Comment:
As we just had the discussion somewhere else: this won't be the way right? How to pin and update it properly?

Reply:
added it now as info :)